news-JS/src/components/controller/Loader/enums/Status.ts

```diff
@@ -0,0 +1,6 @@
+enum Status {
+  Unauthorized = 401,
+  NotFound = 404,
```

Comment:
Do it in lowercase. Like ordinary object

```suggestion
  unauthorized = 401,
  notFound = 404,
```